### PR TITLE
[INJIWEB-1046]: Inji Web Rebrand to InjiWallet in FAQ

### PR DESCRIPTION
**Pull request by kamlesh012**

### CODE REVIEW

File: inji-web/src/locales/kn.json [hunk 1]
```diff
@@ -300,7 +282,6 @@
   },
   "CustomExpiryModal": {
     "title": "ಈ ರುಜುವಾತುಗಳನ್ನು ನೀವು ಎಷ್ಟು ಬಾರಿ ಬಳಸಲು ಬಯಸುತ್ತೀರಿ ಎಂಬುದನ್ನು ದಯವಿಟ್ಟು ನಮೂದಿಸಿ ",
-    "error": "ನಿಮ್ಮ ರುಜುವಾತುಗಳನ್ನು ಎಷ್ಟು ಬಾರಿ ಪರಿಶೀಲಿಸಬಹುದು ಎಂಬುದನ್ನು ಮಿತಿಗೊಳಿಸಲು ದಯವಿಟ್ಟು ಶೂನ್ಯಕ್ಕಿಂತ ಹೆಚ್ಚಿನ ಮಾನ್ಯವಾದ ಮುಕ್ತಾಯ ಸಂಖ್ಯೆಯನ್ನು ನಮೂದಿಸಿ.",
```
Comment (vijay151096) [inline inji-web/src/locales/kn.json:303]: Why is this removed

File: inji-web/src/locales/hi.json [hunk 2]
```diff
@@ -184,100 +184,79 @@
...
-      "description1": "आरंभ करने के लिए, बस इंजी वेब के माध्यम से अपने सत्यापन योग्य क्रेडेंशियल तक पहुंचें।  <a href=\"https://docs.mosip.io/inji/inji-web/functional-overview/end-user-guide\" target=\"_blank\"> उपयोगकर्ता गाइड </a> और <a href=\"https://docs.mosip.io/inji/inji-web/overview\" target=\"_blank\">  प्रलेखन</a>।"
-    },
-    "item23": {
-      "title": "मैं इंजी वेब में एम्बेडेड वीसी और ऑनलाइन शेयरिंग के बीच कैसे टॉगल कर सकता हूं?",
-      "description1": "इंजी वेब में, आप क्यूआर कोड के भीतर पूर्ण सत्यापन योग्य क्रेडेंशियल (वीसी) को एम्बेड करने या मिमोटो जारीकर्ता कॉन्फ़िगरेशन में qr_code_type प्रॉपर्टी को कॉन्फ़िगर करके ऑनलाइन शेयरिंग का उपयोग करने के बीच टॉगल कर सकते हैं। ",
-      "description2": "1. EmbeddedVC: यह विकल्प संपूर्ण वीसी डेटा (JSON प्रारूप में) को सीधे क्यूआर कोड में एम्बेड करता है। ",
-      "description3": "2. OnlineSharing: यह विकल्प एक यूआरएल के साथ एक क्यूआर कोड उत्पन्न करता है जो सुरक्षित रूप से ऑनलाइन संग्रहीत वीसी को इंगित करता है। ",
-      "description4": "ये कॉन्फ़िगरेशन मिमोटो जारीकर्ता के कॉन्फ़िगरेशन में पाए जा सकते हैं। "
-    }
+    "item8":{
```
Comment (vijay151096) [inline inji-web/src/locales/hi.json:187]: Intendation is missing in hi.json

Comment (kamlesh012) [reply inji-web/src/locales/hi.json:187]: missed while transaltion. 
fixed the indentation.

File: inji-web/src/locales/kn.json (hunk 1 above)
Comment (kamlesh012) [reply inji-web/src/locales/kn.json]: missed during translation.re-added it.

**Review by vijay151096 — APPROVED** [review]
lgtm
